cedar-policy-validator/src/typecheck.rs

```diff
@@ -960,6 +969,13 @@ impl<'a> Typechecker<'a> {
                     expr,
                     &[Type::any_entity_reference(), Type::any_record()],
                     type_errors,
+                    |actual| match actual {
+                        Type::Set { .. } => Some(UnexpectedTypeHelp::TryUsingContains),
```

Comment:
what's the rationale for this suggestion?  if I'm understanding correctly, something like `[1, 2, 4].foo` or `[1, 2, 4]["foo"]` will suggest `.contains()`?

Reply:
I was thinking using `[1, 2][0]`, but that doesn't actually parse, so it'd have to be `[1, 2]["0"]`. I'll just remove these.